src/.eslintignore

```diff
@@ -0,0 +1 @@
+.github
```

Comment:
Always add a new line after the end of the file.

Reply:
Done